### PR TITLE
Feat/read write

### PR DESCRIPTION
**Pull request by moarshy**
- adds read/write functionality
- also adds yaml support for passing params
- example yamls and testing.md to test out the different templates

### CODE REVIEW

**Comment by richardblythman** [comment]
force-pushed to rebase with main

File: testing.md [hunk 1]
```diff
@@ -44,6 +44,11 @@ naptha read_storage -id 5io3skxxbwi3ozb5l5rm -o .
 naptha run create_profile_description -f ./example_yamls/describe_daimon_template.yml -l
 ```
 
+### Test `run_ollama`
+```bash
+naptha run run_ollama -f ./example_yamls/run_ollama_template.yml -l
```
Comment (richardblythman) [inline testing.md:49]: I think run_ollama is basically the same as chat right (although the other one uses aiflows). Maybe we can rename them chat_aiflows and chat.

Comment (moarshy) [reply testing.md:49]: aiflows has dependency issue with fastapi/openai modules. i have removed it. but can rename it. if we rename, we will have to rename the repo as well ya

Comment (richardblythman) [reply testing.md:49]: OK let's use this chat, instead of the aiflows one (because of the dependency issues). I'll push it to the Naptha org.

**Pull request merged**